### DISCUSSION TITLE
Награда для суров (ЧЗХ, маринобаф?!?)

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#314] [Ideas]
Если сурв попадает на шип, то его время респауна автоматически падает до 0, что означает мгновенную доп жизнь, после смерти на сурве

**Comment by fe-ed** [comment]
Хммм, довольно странная награда, но то что она ему нужна - это факт

**Comment by lorianss** [comment]
Я бы лучше скилы дал сурвам выше, чем у маров.
Выжил за сурва - играешь за мара+
Думайте

**Reply by Shootnik01** [reply]
Ну тип, попадаешь на корабль, следуюзий спавн за мара с доп скилами
